packages/editor/src/plugins/fill-in-the-blanks-exercise/renderer.tsx

```diff
@@ -78,156 +57,171 @@ export function FillInTheBlanksRenderer(props: FillInTheBlanksRendererProps) {
   }, [blanks, textUserTypedIntoBlanks, initialTextInBlank])
 
   const draggables = useMemo(() => {
-    return blanks.map(({ blankId, correctAnswers }) => ({
+    const sorted = blanks.map(({ blankId, correctAnswers }) => ({
       draggableId: `solution-${blankId}`,
       text: correctAnswers[0].answer,
     }))
-  }, [blanks])
+    if (isEditing) return sorted
```

Comment:
Nice that you kept it sorted in edit mode.

Reply:
Tbh, just asked Gregor what the right way is 😁